sdk/keyvault/azure-keyvault-keys/tests/test_crypto_client_async.py

```diff
@@ -134,15 +137,10 @@ async def _import_symmetric_test_key(self, client, name):
         assert key_vault_key.key.kid == imported_key.id == key_vault_key.id
         return key_vault_key
 
-    @parameterized.expand([param(is_hsm=b) for b in [True, False]], name_func=suffixed_test_name)
-    @PowerShellPreparer("keyvault")
-    async def test_ec_key_id(self, **kwargs):
+    @all_api_versions()
+    @client_setup
+    async def test_ec_key_id(self, key_client, is_hsm, **kwargs):
```

Comment:
`client_setup` passes `is_hsm` by keyword and the test doesn't use it, may as well leave it in kwargs:
```suggestion
    async def test_ec_key_id(self, key_client, **kwargs):
```

Reply:
`is_hsm` is used here as the value of `hardware_protected` when creating a key. I did make sure to leave out the positional parameter in tests that don't use it to keep these nice and tidy 🤓